plugins/colorbutton/plugin.js

```diff
@@ -210,14 +210,14 @@ CKEDITOR.plugins.add( 'colorbutton', {
 				if ( color == '?' ) {
 					editor.getColorFromDialog( function( color ) {
 						if ( color ) {
-							return applyColor( color );
+							return setColor( color );
 						}
 					} );
 				} else {
-					return applyColor( color );
+					return setColor( color );
 				}
 
-				function applyColor( color ) {
+				function setColor( color ) {
 					// Clean up any conflicting style within the range.
 					editor.removeStyle( new CKEDITOR.style( config[ 'colorButton_' + type + 'Style' ], { color: 'inherit' } ) );
 					var colorStyle = config[ 'colorButton_' + type + 'Style' ];
```

Comment:
Since `config[ 'colorButton_' + type + 'Style' ]` is extracted to `colorStyle` we should use it in the line above. So move the var statement to the top of this function.